### PR TITLE
Modify English Doc of ControlFlow APIs for Scalar Tensor

### PR DESCRIPTION
**Pull request by zhhsplendid**
### PR types

Others
### PR changes

Docs
### Describe

Modify English Doc of ControlFlow APIs for Scalar Tensor

Corresponding Chinese Doc PR: https://github.com/PaddlePaddle/docs/pull/5588

### CODE REVIEW

File: python/paddle/static/nn/control_flow.py [hunk 1]
```diff
@@ -569,7 +569,7 @@ def case(pred_fn_pairs, default=None, name=None):
     This operator works like an if-elif-elif-else chain.
 
     Args:
-        pred_fn_pairs(list|tuple): A list or tuple of (pred, fn) pairs. ``pred`` is a boolean Tensor with shape [1], ``fn`` is a callable. All callables return the same structure of Tensors.
+        pred_fn_pairs(list|tuple): A list or tuple of (pred, fn) pairs. ``pred`` is a boolean Tensor whose numel should be 1 (scalar Tensor or shape [1]), ``fn`` is a callable. All callables return the same structure of Tensors.
```
Comment (zhwesky2010) [inline python/paddle/static/nn/control_flow.py]: shape [] or shape [1] 吧

File: python/paddle/static/nn/control_flow.py [hunk 2]
```diff
@@ -702,7 +702,7 @@ def switch_case(branch_index, branch_fns, default=None, name=None):
     This operator is like a C++ switch/case statement.
 
     Args:
-        branch_index(Tensor): A Tensor with shape [1] to specify which branch to execute. The data type is ``int32``, ``int64`` or ``uint8``.
+        branch_index(Tensor): A Tensor whose numel should be 1 (scalar Tensor or shape [1]) to specify which branch to execute. The data type is ``int32``, ``int64`` or ``uint8``.
```
Comment (zhwesky2010) [inline python/paddle/static/nn/control_flow.py]: shape [] or shape [1] 吧

File: python/paddle/static/nn/control_flow.py [hunk 3]
```diff
@@ -910,9 +910,9 @@ def cond(pred, true_fn=None, false_fn=None, name=None, return_names=None):
         branch will be executed during runtime.
 
     Args:
-        pred(Tensor): A boolean tensor whose numel should be 1. The boolean
-            value determines whether to return the result of ``true_fn`` or
-            ``false_fn`` .
+        pred(Tensor): A boolean tensor whose numel should be 1 (scalar Tensor
+            or shape [1]). The boolean value determines whether to return the
```
Comment (zhwesky2010) [inline python/paddle/static/nn/control_flow.py:914]: 同上

File: python/paddle/static/nn/control_flow.py (hunk 1 above)
Comment (zhhsplendid) [reply python/paddle/static/nn/control_flow.py]: Done

File: python/paddle/static/nn/control_flow.py (hunk 2 above)
Comment (zhhsplendid) [reply python/paddle/static/nn/control_flow.py]: Done

File: python/paddle/static/nn/control_flow.py (hunk 3 above)
Comment (zhhsplendid) [reply python/paddle/static/nn/control_flow.py:914]: Done

**Review by zhwesky2010 — APPROVED** [review]
LGTM

**Review by Ligoml — APPROVED** [review]
LGTM for docs

**Pull request merged**